ui/app/rif/components/luminoNetworkChannels.js

```diff
@@ -106,7 +106,7 @@ class LuminoNetworkChannels extends Component {
         </div>
       }
         {isOwner &&
-          <OpenChannel/>
+          <OpenChannel afterChannelCreated={this.loadChannels()}/>
```

Comment:
Wont this be executed only once? I mean you could pass this.loadChannels (without the parenthesis)

or () => this.loadChannels()

Reply:
it's always better to use () => sintax